doc/stdenv/platform-notes.chapter.md

```diff
@@ -84,24 +84,19 @@ Some common issues when packaging software for Darwin:
   stdenv.mkDerivation {
     name = "libfoo-1.2.3";
     # ...
-    env.PACKAGE_SPECIFIC_SDK_VAR = apple-sdk_10_15.sdkroot;
+    env.PACKAGE_SPECIFIC_SDK_VAR = apple-sdk_12.sdkroot;
     # or
-    buildInputs = [ apple-sdk_10_15 ];
+    buildInputs = [ apple-sdk_12 ];
     postPatch = ''
       export PACKAGE_SPECIFIC_SDK_VAR=$SDKROOT
     '';
   }
   ```
 
-  The following is a list of Xcode versions, the SDK version in nixpkgs, and the attribute to use to add it. Generally, only the last SDK release for a major version is packaged (each _x_ in 10._x_ until 10.15 is considered a major version).
+  The following is a list of Xcode versions, the SDK version in nixpkgs, and the attribute to use to add it. Generally, only the last SDK release for a major version is packaged.
 
   | Xcode version      | SDK version                                       | nixpkgs attribute |
   |--------------------|---------------------------------------------------|-------------------|
-  | Varies by platform | 10.12.2 (x86_64-darwin)<br/>11.3 (aarch64-darwin) | `apple-sdk`       |
```

Comment:
Should the entry for the bare `apple-sdk` be in this table, just to be clear what does it do after the change?

Reply:
I’ve added the `apple-sdk` reference to the same table entry as `apple-sdk_11` now that it’s not platform-specific; let me know what you think!

Reply:
perfect ❤️